### PR TITLE
Maintenance mode interface.

### PR DESCRIPTION
**Pull request by mickel1138**

### CODE REVIEW

File: src/Pelagos/Bundle/AppBundle/Controller/UI/MaintenanceModeController.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+     *
+     * @param Request $request The Symfony request object.
+     *
+     * @Route("/")
+     *
+     * @return Response A Response instance.
+     */
+    public function defaultAction(Request $request)
+    {
+        $this->denyAccessUnlessGranted('IS_AUTHENTICATED_FULLY');
```
Comment (praneethpr) [inline src/Pelagos/Bundle/AppBundle/Controller/UI/MaintenanceModeController.php]: I think we should give access only to DRPMs

File: src/Pelagos/Bundle/AppBundle/Resources/views/MaintenanceMode/maintenanceMode.html.twig [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+<label for="bannertext">Banner Text:<label>
+<input type="text" size="200" id="bannertext" name="bannertext" placeholder="Your Banner Text Here" required value="{{ getMaintenanceModeText() }}">
+<label for="bannercolor">Banner Color:<label>
+<input type="color" id="bannercolor" name="bannercolor" value="{{ '#EED202'|maintenanceModeColor }}">
+<label for="bannermode">Mode:</label>
+<input type="radio" id="bannermodeon" name="bannermode" value="activate" {{isMaintenanceMode() ? 'checked' : ''}}>Active
+<input type="radio" id="bannermodeoff" name="bannermode" value="deactivate" {{not isMaintenanceMode() ? 'checked' : ''}}>Off<br>
+<input type="submit">
+
+{% endblock %}
```
Comment (praneethpr) [inline src/Pelagos/Bundle/AppBundle/Resources/views/MaintenanceMode/maintenanceMode.html.twig]: New line at the end

File: src/Pelagos/Bundle/AppBundle/Twig/Extensions.php [hunk 3]
```diff
@@ -243,6 +247,24 @@ public function getMaintenanceModeColor() : ? string
...
     }
 
+    /**
+     * Gets maintenance mode color.
+     *
+     * @param string $color The color text.
+     *
+     * @return string|null Returns maintenance mode banner color.
+     */
+    public function maintenanceModeColor(string $color) : string
```
Comment (praneethpr) [inline src/Pelagos/Bundle/AppBundle/Twig/Extensions.php]: We have two functions in this class which gets the maintenance mode color?

File: src/Pelagos/Bundle/AppBundle/Controller/UI/MaintenanceModeController.php (hunk 1 above)
Comment (mickel1138) [reply src/Pelagos/Bundle/AppBundle/Controller/UI/MaintenanceModeController.php]: Yes, have to change that.

File: src/Pelagos/Bundle/AppBundle/Twig/Extensions.php (hunk 3 above)
Comment (mickel1138) [reply src/Pelagos/Bundle/AppBundle/Twig/Extensions.php]: One of them is a filter, but this one can call the other one.

**Pull request merged**